components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -211,60 +217,93 @@ function TeamUsage() {
                                             >
-                                                Started
+                                                Timestamp
                                                 <SortArrow
-                                                    className={`h-4 w-4 my-auto ${
+                                                    className={`ml-2 h-4 w-4 my-auto ${
                                                         isStartedTimeDescending ? "" : " transform rotate-180"
                                                     }`}
                                                 />
                                             </span>
                                         </ItemField>
                                     </Item>
                                     {currentPaginatedResults &&
-                                        currentPaginatedResults.map((usage) => (
-                                            <div
-                                                key={usage.instanceId}
-                                                className="flex p-3 grid grid-cols-5 justify-between transition ease-in-out rounded-xl focus:bg-gitpod-kumquat-light"
-                                            >
-                                                <div className="my-auto">
-                                                    <span>{getType(usage.workspaceType)}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-400">{usage.workspaceClass}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-700">{getMinutes(usage)}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-700">{usage.credits.toFixed(1)}</span>
-                                                </div>
-                                                <div className="my-auto">
-                                                    <span className="text-gray-400">
-                                                        {displayTime(usage.startTime)}
-                                                    </span>
+                                        currentPaginatedResults.map((usage) => {
+                                            return (
+                                                <div
+                                                    key={usage.instanceId}
+                                                    className="flex p-3 grid grid-cols-12 gap-x-3 justify-between transition ease-in-out rounded-xl dark:bg-gray-800"
```

Comment:
**nitpick**: Sorry, I meant to suggest this for the table header, maybe I placed the comment on the wrong line of code?

```suggestion
                                                    className="flex p-3 grid grid-cols-12 gap-x-3 justify-between transition ease-in-out rounded-xl"
```

Reply:
Posting some screenshots for more context.

| Light Theme | Dark Theme |
|-|-|
| <img width="1440" alt="Screenshot 2022-08-18 at 8 41 05 PM" src="https://user-images.githubusercontent.com/120486/185460016-534017f7-27fa-4983-b840-f4db8d7d9595.png"> | <img width="1440" alt="Screenshot 2022-08-18 at 8 41 14 PM" src="https://user-images.githubusercontent.com/120486/185460021-735dc660-af07-424c-b26d-7e65e57959f5.png"> |

Hover Issue on Dark Theme: 

https://user-images.githubusercontent.com/120486/185460202-e2c90905-adae-4748-a066-87bbbcdc4a45.mov

Reply:
@geropl The table header color on the dark theme is still unresolved. It needs to use bg-gray-800 without any color changes on hover.

Reply:
Resolving as this has been fixed.
Thanks @geropl! Just realized you are fixing this now and changes have not been pushed yet.

Reply:
:100: :laughing: